package_manifest.yaml

```diff
@@ -29,9 +29,52 @@ all:
         dist: '.el8'
         macros:
           foremandist: "{{ foremandist }}"
+    copr_project_user: "@theforeman"
   children:
     packages: {}
     repoclosures: {}
+    copr_projects: {}
+
+copr_projects:
+  hosts:
+    foreman-copr:
+      copr_project_name: "foreman-{{ foreman_version }}-staging"
+      copr_project_chroots:
+        - name: rhel-8-x86_64
+          modules:
+            - 'ruby:2.7'
+            - 'python39:3.9'
+            - 'nodejs:12'
+          external_repos:
+            - https://yum.puppet.com/puppet7/el/8/x86_64/
+    plugins-copr:
+      copr_project_name: "foreman-plugins-{{ foreman_version }}-staging"
+      copr_project_chroots:
+        - name: rhel-8-x86_64
+          modules:
+            - 'ruby:2.7'
+            - 'python39:3.9'
+            - 'nodejs:12'
+          external_repos:
+            - https://download.copr.fedorainfracloud.org/results/@theforeman/foreman-nightly-staging/rhel-8-x86_64/
```

Comment:
copr can refer to other coprs using `copr://user/project` syntax -- no idea if that's better or worse, but wanted to point it out :)